Client/Frontend/Shields/ShareShieldsController.swift

```diff
@@ -0,0 +1,278 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import UIKit
+import BraveShared
+import Shared
+import BraveUI
+
+// MARK: - ShareShieldsViewController
+
+class ShareShieldsViewController: UIViewController, Themeable {
+
+    // MARK: Action
+    
+    enum Action {
+        case shareEmailClicked
+        case shareTwitterClicked
+        case shareFacebookClicked
+        case shareDefaultClicked
```

Comment:
Add a comment what `default` share action is about, I assume the generic iOS share menu?

Reply:
Yes that one will bring generic menu. 
Comment is added.